### ISSUE TITLE
Fine-tuning just one class from COCO

### ISSUE THREAD

**Issue by ekarabulut** [AlexeyAB/darknet#4585]
Hi Alexey,

Before taking your opinion about my issue, I've searched the issue database and tried to understand what you previously suggested. So, I've read #2139, #3719, #2147. In general, they asked how to update COCO pretrained weights with new classes or improve just one class while keeping the others.

My question is a bit different. I just want to take COCO pretrained weight files and produce a new detection model just for one class (like "person") from COCO. 

What is the best way to accomplish this task with a good accuracy?

I'd be glad if you can share your opinion.

**Comment by AlexeyAB** [comment]
* Or add `dont_show` before each class except `person` in `coco.names` file. Just run detection by using default yolov3.weights without training
* Or remove all classes from COCO dataset, and keep only `person`, then train your own model

**Comment by ekarabulut** [comment]
Thank you for the quick reply.

As you suggested, I plan to start a training by using COCO dataset combined with my own images.

In this case, is it a good idea to use yolov3.weights   to start training? In some posts it is suggested to use a modified version of this file by using "partial" command.

**Comment by AlexeyAB** [comment]
Use yolov3.conv.81
https://github.com/AlexeyAB/darknet/blob/master/build/darknet/x64/partial.cmd#L27

**Comment by ekarabulut** [comment]
Thanks @AlexeyAB. It really helped me.

**Comment by Turkishvein** [comment]
> Use yolov3.conv.81 https://github.com/AlexeyAB/darknet/blob/master/build/darknet/x64/partial.cmd#L27

Why 81, not 82 or something?
